src/main/scala/uk/gov/hmrc/ct/box/Validators.scala

```diff
@@ -19,69 +19,69 @@ package uk.gov.hmrc.ct.box
 import uk.gov.hmrc.ct.box.retriever.BoxRetriever
 import uk.gov.hmrc.ct.domain.ValidationConstants._
 
-trait Validators[B <: BoxRetriever] {
+trait Validators {
 
   protected val boxId = getClass.getSimpleName
 
-  protected def And(predicates: ((B) => Boolean)*)(boxRetriever: B): Boolean = {
-    !predicates.exists { p => !p(boxRetriever)}
+  protected def And(predicates: (() => Boolean)*)(): Boolean = {
+     !predicates.exists { p => !p()}
```

Comment:
actually I think this is better:
protected def And(predicates: (() => Boolean)*)(): Boolean = {
     predicates.forall {_()}
  }

  protected def Or(predicates: (() => Boolean)*)(): Boolean = {
     !predicates.forall {!_()}
  }